dev-python/requests-futures/requests_futures-1.0.0.recipe

```diff
@@ -23,7 +23,7 @@ BUILD_REQUIRES="
 	"
 
 PYTHON_PACKAGES=(python python3)
-PYTHON_VERSIONS=(2.7 3.7)
+PYTHON_VERSIONS=(2.7 3.9)
```

Comment:
it's probably a good idea to remove python2 as well ("2.7" here and "python" on the line above)

```suggestion
PYTHON_VERSIONS=(3.9)
```